src/i18n/es.js

```diff
@@ -34,20 +34,47 @@ const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
 
 export default {
   change_plan: {
+    agencies_icon: `https://cdn.fromdoppler.com/doppler-ui-library/${dopplerUiLibraryVersion}/img/icono-agencias.svg`,
     all_of_plan: `Todo lo de {plan} más:`,
+    ask_demo: 'Solicita una demo',
+    big_data_tooltip: 'Funcionalidad Big Data',
     calculate_price: 'Calcular Precio',
     cancel_campaign: 'Cancelar Campaña',
     compare_features: 'Comparar Funcionalidades',
+    current_plan: 'Plan Actual',
     description: '¿Realizas envíos periodicamente o un newsletter al mes?',
+    description_agencies: 'Gestiona y monitorea las cuentas de tus clientes desde un solo lugar.',
+    description_free: 'Prueba Doppler GRATIS sin contratos ni tarjetas de crédito. Hasta 500 Contactos.',
+    description_plus: 'Funciones avanzadas para profesionales que necesitan mas personalización',
+    description_standard: 'Prueba nuestros planes que se ajusten mas a tus necesidades: mensuales o prepagos.',
     email_parameter: 'Parámetros de Email',
-    features: 'Funcionalidades:',
+    features_HTML_agencies: `<option>Administración de varias cuentas</option>
+    <option>Administración de permisos</option>
+    <option>Administración de roles</option>`,
+    features_HTML_free: `<option>Atención al Cliente</option>
+    <option>Campañas Convencionales</option>
+    <option>Campañas Automation</option>`,
+    features_HTML_plus: `<star>Atención preferencial</star>
+    <newOption>Detener Campañas</newOption>
+    <option>Landing pages con dominio personalizado</option>
+    <newBigData>Recomendación de Asuntos</newBigData>`,
+    features_HTML_standard: `<option>Acceso a la API de Doppler</option>
+    <option>Campañas Automation de Comportamiento en Sitio</option>
+    <option>Envíos con tu firma personalizada</option>`,
```

Comment:
En general en los contenidos multilinea los estabamos comenzando junto al margen con una interlinea anterior:

```suggestion
    features_HTML_standard: `
<option>Acceso a la API de Doppler</option>
<option>Campañas Automation de Comportamiento en Sitio</option>
<option>Envíos con tu firma personalizada</option>`,
```

Ejemplo:
![image](https://user-images.githubusercontent.com/1157864/93467718-6b8e7c80-f8c4-11ea-8360-9d2a4022f152.png)

Reply:
Pero esos son markup y mis textos son un casi html, estaría bien igual?

Reply:
creo que si